packages/modal/cypress/integration/modal.test.js

```diff
@@ -0,0 +1,64 @@
+const { cy, describe, it } = global;
+
+describe('Modal TCs', () => {
+  it('The correspondent modal title and content is displayed', () => {
+    cy.fixture('modal.js').then((modal) => {
+      return cy
+        .visit('/')
+        .window()
+        .then((win) => {
+          win.modalPropsSubject.next({
+            ...modal,
+            isOpen: true,
+          });
+
+          return cy.modalOpenend(modal);
+        });
+    });
+  });
+
+  it('Check default value of isOpen property', () => {
+    cy.fixture('modal.js').then((modal) => {
+      return cy.visit('/').modalClosed(modal);
+    });
+  });
+
+  it('Click on cross - isOpen became false', () => {
+    cy.fixture('modal.js').then((modal) => {
+      return cy
+        .visit('/')
+        .window()
+        .then((win) => {
+          win.modalPropsSubject.next({
+            ...modal,
+            isOpen: true,
+          });
+
+          return cy
+            .get('#web-modal')
+            .find('[data-cy=modal-close]')
```

Comment:
```suggestion
            .get('#web-modal [data-cy=modal-close]')
```

So you don't need .get('#web-modal')